README.md

```diff
@@ -1 +1,8 @@
 # Travis and Kayla's Super Awesome Repo
+
+## Setup Instructions
+1. Clone down the repo
+```
+git clone git@github.com:kaylagordon/git-practice-2201.git
+```
+2. Check out our amazing README
```

Comment:
Maybe we should add a step here for how to open the file.

Reply:
@Kalikoze can you add a step here to show how to open the file?

Reply:
Great suggestion Kayla, thank you!